src/imex.rs

```diff
@@ -386,7 +428,11 @@ async fn imex_inner(
     let success = match what {
         ImexMode::ExportSelfKeys => export_self_keys(context, path).await,
         ImexMode::ImportSelfKeys => import_self_keys(context, path).await,
-        ImexMode::ExportBackup => export_backup(context, path).await,
+
+        // TODO In some months we can change the export_backup_old() call to export_backup() and delete export_backup_old().
+        // (now is 07/2020)
```

Comment:
hum, i think it's ok to remove export_backup_old and these comments here completely now. 
This means that you can't import new-format backup data in older apps.  But this is by far less of a problem than the RSA2048->EDD25519 key transition we need.  My own exporting/importing does not affect anybody else in the eco-system.  It's anyway a good idea to see to get a couple of new releases out before end july which is pretty soon.

Reply:
Ok.